timon/probes.py

```diff
@@ -351,17 +352,18 @@ def parse_result(self, jsonstr):
         logger.debug("resp %r", resp)
 
         exit_code = rslt['exit_code']
-        if exit_code == flags.FLAG_UNKNOWN:
-            return
         self.msg = resp.get('msg') or rslt.get('reason') or ''
-        if rslt.get("status") != 200:
-            self.status = " UNKNOWN"
+        if exit_code != flags.FLAG_OK:
+            self.status = flags.INV_FLAG_MAP.get(exit_code, "ERROR")
+            return
         elif self.match_rule(rslt, self.ok_rule):
             self.status = "OK"
         elif self.match_rule(rslt, self.warning_rule):
             self.status = "WARNING"
         elif self.match_rule(rslt, self.error_rule):
             self.status = "ERROR"
+        else:
```

Comment:
If there is match rule for errors and it is explicitly not matched, why should the state become `"ERROR"`
This is what the error rule is for.

Reply:
Klaus, yesterday, in conf call, we agree that unknown status is normally only for 404 right?

Reply:
not really. 
Why is there an error rule if you ignore it.

Basically your change means, that you  suggest to remove `error_rule` because anything, that is not OK or warning will become ERROR

Reply:
The idea to have a different default val which will be configurable is a good idea i remove the else, but I think unknown is not a good idea in default val for each strange case

Reply:
if it is configurable we can set the default to error :-)